src/mmrelay/meshtastic_utils.py

```
@@ -4587,9 +4708,15 @@ async def reconnect() -> None:
                 loop = asyncio.get_running_loop()
                 # Pass the current config during reconnection to ensure matrix_rooms is populated
                 # Using None for passed_config would skip matrix_rooms initialization
-                meshtastic_client = await loop.run_in_executor(
+                connect_future = loop.run_in_executor(
                     None, connect_meshtastic, config, True
```

Ruff (0.15.9)

[warning] Boolean positional value in function call

(FBT003)